src/MediaPlayer/Plyr/Addons/SharedFeatures/SharedFeatures.ts

```diff
@@ -0,0 +1,13 @@
+import './SharedFeatures.less';
+export class CreateOverlay {
+  public static containerExists = plyrContainer => {
+    let overlay = null;
+    if (document.getElementById('overlay') !== null) {
+      overlay = document.getElementById('overlay');
+    } else if (document.getElementById('overlay') === null) {
```

Comment:
This could be just an else, instead of an else if?

Also, it could be fun to try and write a test for this class, up to you though